test/src/main/java/org/apache/accumulo/test/SplitCancelsMajCIT.java

```diff
@@ -76,16 +76,13 @@ public void test() throws Exception {
       }
       // start majc
       final AtomicReference<Exception> ex = new AtomicReference<>();
-      Thread thread = new Thread() {
-        @Override
-        public void run() {
-          try {
-            c.tableOperations().compact(tableName, null, null, true, true);
-          } catch (Exception e) {
-            ex.set(e);
-          }
+      Thread thread = new Thread(() -> {
+        try {
+          c.tableOperations().compact(tableName, null, null, true, true);
+        } catch (Exception e) {
+          ex.set(e);
```

Comment:
Clever way to track exceptions inside the thread. Do we want to do this elsewhere?

Reply:
I looked around and didn't see anywhere else where this should be used.